src/process/process_set.cu

```diff
@@ -106,6 +108,11 @@ namespace micm
           react_ids_offset += d_number_of_reactants[i_rxn];
           yields_offset += d_number_of_products[i_rxn];
         }  // end of loop over reactions in a grid
+        // Form the minus Jaocbian matrix (-J)
+        for (size_t idx = 0; idx < num_elments_per_jacobian; ++idx)
+        {
+            d_jacobian[idx*num_grid_cells + tid] = -d_jacobian[idx*num_grid_cells + tid]; 
+        }
```

Comment:
Thanks @mwaxmonsky . I agree that moving this section to `AlphaMinusJacobianKernel` function will avoid changing the test for the `AddJacobianTerms` function. The reason I did not do it is that negating Jacobian in `AlphaMinusJacobianKernel` function will increase the resource request (i.e., we need to go through the whole Jacobian matrix rather than the diagonal elements only) and reduce the parallelism (i.e., each CUDA thread needs to loop through the `jacobian_diagonal_elements` in serial and checks if it is working on a diagonal element), compared to current implementation. In contrast, the `AddJacobianTerms` just parallelizes at the `num_grid_cells` level and all the other loops are done in serial. Thus negating the Jacobian matrix here does not request more resources and reduce the parallelism. I am happy to make the changes you suggested and we can revisit this implementation if they become a hotspot in the future performance tests.

Reply:
Hmmmmm, @mattldawson, is this a reasonable tradeoff?  We would bifurcate the meaning of the API but gain the efficiency we expect for GPUs.